components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5944,19 +5982,32 @@ def memberInsert(memberInfo):
                                       (name, name, varInsert(name, member.identifier.name).define()))
             return CGGeneric("%s\n" % insertion.define())
 
-        memberInits = CGList([memberInit(m) for m in self.memberInfo])
         memberInserts = CGList([memberInsert(m) for m in self.memberInfo])
 
+        selfName = self.makeClassName(d)
+        if self.membersNeedTracing():
+            actualType = "RootedTraceableBox<%s>" % selfName
+            preInitial = "let mut dictionary = RootedTraceableBox::new(%s::default());\n" % selfName
```

Comment:
do you need the interpolation? I'd expected `RootedTraceableBox::default()` to just work.